Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -168,33 +167,31 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         serializedDiagnostics: Bool = false,
         isManifestSandboxEnabled: Bool = true,
         cacheDir: AbsolutePath? = nil,
+        useInMemoryCache: Bool = true,
         delegate: ManifestLoaderDelegate? = nil,
         extraManifestFlags: [String] = []
-    ) {
+    ) throws {
```

Comment:
Since this will cause all clients to have to adjust, I suggest adding `useInMemoryCache` as a default-less argument, and adding the old initializer as a deprecated convenience initializer that doesn't throw.  This will give clients time to adjust.

Reply:
yup. reverted to non-throwing as there is a different way